packages/contracts-bedrock/CounterNames.json

```diff
@@ -0,0 +1,4 @@
+{
+  "0x5615dEB798BB3E4dFa0139dFa1b3D433Cc23b72f" : "Counter",
+  "0x2e234DAe75C793f67A35089C9d99245E1C58470b" : "Counter2"
+}
```

Comment:
What is this file for?

Reply:
That file is for testing the summarization feature with the `Counter` contract. It should be removed before merging, but for the time being, I decided to include the `Counter` files in case we want to do some more testing